### PR TITLE
Update install.rst

### PR DESCRIPTION
**Pull request by bencipher**
Updated steps for installing and configuring sekizai

## Related resources

* #...
* #...

## Checklist

* [x] I have opened this pull request against ``develop``
* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on [Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

File: docs/how_to/install.rst [hunk 1]
```diff
@@ -210,16 +210,15 @@ After each of the steps below run ``cms check`` to verify that you have resolved
 Sekizai
 =======
 
-`Django Sekizai <https://github.com/ojii/django-sekizai>`_ is required by the CMS for static files management. You need
-to have::
+`Django Sekizai <https://github.com/ojii/django-sekizai>`_ is required by the CMS for static files management. For that, you need to install the package, add it to installed apps and to your context_processors by following steps::
```
Comment (marksweb) [inline docs/how_to/install.rst:213]: Sekizai is one of the requirements of the cms so we shouldn't need to specify installing it separately.

See here: https://github.com/django-cms/django-cms/blob/develop/setup.py#L13

**Review by marksweb — COMMENTED** [review]
I'm not sure this change is required. The package is installed as a dependency.

The syntax of these changes is also incorrect as you csn see in the test failures.

**Comment by bencipher** [comment]
Yes you are correct about not needing to explicitly install the Selizai dependency. I think the doc was not formatted for clear interpretation at the configuration of sekizai. I could remove the PR or just remove the installation script for the Sekizai library, your call.

File: docs/how_to/install.rst (hunk 1 above)
Comment (bencipher) [reply docs/how_to/install.rst:213]: Yes, I can remove the install line. I think for some reasons, it wasn’t installed for me and I had to manually install it and packaging library which was why I added it in the docs

**Pull request closed without merging**